modules/crafted-updates-config.el

```diff
@@ -10,16 +10,9 @@
 ;; Checks for updates to the Crafted Emacs project.  Provides a
 ;; function to show the updates before pulling changes.
 
-;; FIXME: [2023-01-31 Tue] This no longer works.  The problem is we
-;; don't know where the project was cloned to.  Formerly it was
-;; `user-emacs-directory', however, now we allow the user to clone
-;; this project wherever, add it (ostensibly the modules folder) to
-;; the `load-path' and then build their configuration however they see
-;; fit.  Unless we set a variable like `crafted-emacs-home' there is
-;; no good way to know where this project is cloned.
-
-;; UPDATE: [2023-02-09 Thu] `crafted-emacs-home' is now set on load,
-;; this can be updated to use that value for checking on updates from
+;; This uses `crafted-emacs-home', which is set on load by if it hasn't been
```

Comment:
I might be reading this wrong, but there is a "by" too much?

```
This uses `crafted-emacs-home', which is set on load if it hasn't been 
set by the user or `crafted-init-config'.
```

Reply:
Oh yes, that's right. Good eye, thank you!